_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -1056,7 +1066,12 @@ mfxStatus VAAPIVideoProcessing::Execute(mfxExecuteParams *pParams)
     output_region.width  = outInfo->CropW;
     m_pipelineParam[0].output_region = &output_region;
 
-    m_pipelineParam[0].output_background_color = 0xff000000; // black for ARGB
+    m_pipelineParam[0].output_background_color = RGBA64TO32(pParams->iBackgroundColor);
+
+    // Do not disable colorfill if letterboxing is used
```

Comment:
PLEASE, no more of this CRAP!!

You have uAPI to enable/disable color fill. I wish to see these 3 lines of code GONE forever!